packages/typescriptlang-org/src/lib/release-info.json

```diff
@@ -19,12 +19,12 @@
       "vs2019_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40rc"
     },
     "beta": {
-      "vs2017_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40beta",
-      "vs2019_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40beta"
-    },
-    "rc": {
       "vs2017_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40rc",
       "vs2019_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40rc"
+    },
+    "rc": {
+      "vs2017_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40beta",
+      "vs2019_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40beta"
     }
   }
 }
```

Comment:
If you remove the changes to this file (I'm unsure what changed to be honest) tenths POR can be auto-merged by the folks running the Portuguese language

Reply:
Adjusted